scripts/server

```diff
@@ -9,16 +14,18 @@ Starts servers using docker-compose
 "
 }
 
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
         usage
+    elif [ "${1:-}" = "--java" ]; then
```

Comment:
I haven't experienced any issues running Raster Foundry via the `sbt run` task. I think this should be the default behavior we target, and if we want to run the assembly jars, we should consider that an override of the default behavior.

Reply:
I think this is a risky change for development. The development team's experience has been the opposite as of recently - we prefer running directly from the jars for a few reasons:

 - We've found `sbt` sometimes unnecessarily compiles subprojects in a non-predictable manner which slows server restart time and often results in having to recompile both the `api` and `backsplash-server` subproject when we're only testing one -- so `assembly` turns out to be much quicker
 - When debugging threading behavior in the API or tile server, running through `sbt` makes this difficult to debug because of the way `sbt` manages its own threadpool
 - Running from the assembly jars better approximates how these services are deployed

I'd be in favor of making running from the assembly jars the default - it's been the de facto default for the last 2 months without much, if any issue

Reply:
Thank you for sharing that context. I do think it makes more sense to use `assembly` given these challenges.